### ISSUE TITLE
Scrollbar if no edit or delete action

### ISSUE THREAD

**Issue by sekru1** [dcasia/icon-action-toolbar#20]
I have noticed that the action toolbar in the detail view gets a scrollbar if the edit and delete button is missing (e.g. due to missing rights).
This looks very ugly.
(It is not a question of the number of buttons.)
When I activate one of the standard actions, the scrollbar disappears .
Is it possible to fix this bug?

![grafik](https://github.com/user-attachments/assets/498d7ad4-fcd2-4342-94f1-47431f284aad)

**Comment by samyouel** [comment]
Same thing here...
If I leave the label empty, it even disappears completely:
```php
Actions\MyActionName::make()->icon('pencil', label: ''),
```
<img width="538" alt="image" src="https://github.com/user-attachments/assets/b04a7b55-5fb1-49d2-8506-f24ae319df63">

**Comment by damarinov1** [comment]
Yep, same problem here as well. I've made a temporary fix for now. I've overridden the class responsible in my admin styles.

```css
.icon-action-toolbar-wrapper {
  overflow-x: visible !important;
}
```

OR you can downgrade to v0.1.2 , I see that this issue was introduced in v0.1.3

**Comment by scramatte** [comment]
+1
